src/qt/bitcoingui.cpp

```diff
@@ -537,23 +536,6 @@ void qtSetSessionInfo(std::string defaultgrcaddress, std::string cpid, double ma
 	#endif
 }
 
-std::string qtGRCCodeExecutionSubsystem(std::string sCommand)
```

Comment:
This one actually seemed to do something. I don't know if it's something we need to keep though. @gridcoin?